Svc/FpySequencer/FpySequencerDirectives.cpp

```diff
@@ -418,6 +426,47 @@ Signal FpySequencer::storeTlmVal_directiveHandler(const FpySequencer_StoreTlmVal
     return Signal::stmtResponse_success;
 }
 
+Signal FpySequencer::pushTlmValAndTime_directiveHandler(const FpySequencer_PushTlmValAndTimeDirective& directive,
+                                                        DirectiveError& error) {
+    if (!this->isConnected_getTlmChan_OutputPort(0)) {
+        error = DirectiveError::TLM_GET_NOT_CONNECTED;
+        return Signal::stmtResponse_failure;
+    }
+
+    Fw::Time tlmTime;
```

Comment:
I feel like I have seen this code in the tlm directive before.  Could this be a helper, where we just include the extra bit of time processing?